presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -292,7 +315,21 @@ private MappedAggregationInfo createAggregationOverNull(AggregationNode referenc
                 AggregationNode.Step.SINGLE,
                 Optional.empty(),
                 Optional.empty());
-        return new MappedAggregationInfo(aggregationOverNullRow, aggregationsSymbolMapping);
+
+        return Optional.of(new MappedAggregationInfo(aggregationOverNullRow,
+                                aggregationsSymbolMapping));
+    }
+
+    private static boolean checkSymbolDependency(AggregationNode.Aggregation aggregation, Map<Symbol, SymbolReference> sourcesSymbolMapping)
```

Comment:
Please rename this method to something `isUsingSymbols` or `dependsOn`. Then return value to match the name.